src/main/java/org/example/TicTacToe.java

```diff
@@ -0,0 +1,66 @@
+package org.example;
+
+import java.util.Scanner;
+import java.util.Set;
+
+
+public class TicTacToe {
+    private static final String INPUT = "Input command";
+    private static final String BAD_PARAMETERS = "Bad parameters!";
+    public static void main(String[] args) {
+        menu();
+    }
+
+    public static void menu() {
+
+        Matrix matrix = new Matrix();
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println(INPUT);
+        String input = sc.nextLine();
+        String[] strings = input.split(" ");
+
+        if (input.equals("exit")) System.exit(0);
+        if (strings.length != 3) {
+            System.out.println(BAD_PARAMETERS);
+            menu();
+        }
+
+        Player player1;
+        Player player2;
+
+        if (strings[0].equals("start") &&
+                Set.of("user", "easy", "medium", "hard").contains(strings[1]) &&
+                Set.of("user", "easy", "medium", "hard").contains(strings[2])) {
+
+            player1 = Player.createPlayer(strings[1], 'X');
```

Comment:
X and O could also be constants. Imagine tomorrow someone tells us to use other symbols instead of X and O. It will be so easy to only update constant values.